lib/smart_answer_flows/simplified-expenses-checker/outcomes/you_can_use_result.govspeak.erb

```diff
@@ -1,4 +1,22 @@
 <% content_for :body do %>
+  <% if capital_allowance_claimed %>
+    ##For your vehicle
+
+    You can’t use simplified expenses for your vehicle because you’ve already claimed capital allowances for it.
+
+    You have to continue to calculate your vehicle expenses by working out the actual costs.
+
+    ##For your other business expenses
+
+  <% elsif simplified_expenses_claimed %>
+    ##For your vehicle
+    Based on your answers you would probably be better off [using simplified expenses/working out your expenses based on the actual costs].
+
+    You can’t claim capital allowances for your vehicle because you’ve already claimed simplified expenses for it.
+
+    ##For your other business expenses
```

Comment:
It is also worth stating that my aim thus far is to avoid altering SmartAnswer's State object.. and most of the code you refer to have not been directly touched by this excercise.. I share your concerns about eliminating the precalculate block.. but like I discussed with you.. these will be replaced once we are in a good position to do so..